fe/src/shared/constants/FilesConstants.js

```diff
@@ -1,5 +1,5 @@
 export default class FilesConstants {
     static defaultPageSize() {
-        return 5;
+        return 100;
```

Comment:
**suggestion**: could you please move this to the C# constants and inject it via props? Lets keep up with the pagination conventions we already have in many placek in the platform.